### DISCUSSION TITLE
Move TRMM Agent to another TRMM Server

### DISCUSSION THREAD

**Discussion by mbr-a** [amidaware/tacticalrmm#1569] [Q&A] [answered]
Hi,
Is it a way to move an agent to another TRMM server without uninstall / reinstall agent ?
What's append also for third party component like Mesh and Chocolatey ?

Needs are / can be : 
- Switch agent from an integration server to a prod server
- In case of big software change, prefer to build a new server and migrate progressively

Thanks a lot for your answers.

Regards,

**Comment by dinger1986** [comment, marked as answer]
not without unisntall/reinstall, theres tokens etc created when you create and install on your server

**Reply by mbr-a** [reply]
Thanks for your confirmation.
